.history/background_20250905192227.js

```
@@ -0,0 +1,397 @@
+    const haveStream = typeof getGlobalFunction === 'function' && getGlobalFunction('streamCerebrasCompletion');
+    const haveNonStream = typeof getGlobalFunction === 'function' && getGlobalFunction('getCerebrasCompletion');
+    if (haveStream || haveNonStream) return { haveStream: !!haveStream, haveNonStream: !!haveNonStream };
+    try {
+        // Prefer importScripts when available (classic worker style)
+        if (typeof importScripts === 'function') {
+            importScripts('chat.js');
+        } else if (typeof browser !== 'undefined' && browser.runtime?.getURL) {
+            const url = browser.runtime.getURL('chat.js');
+            // Fallback: fetch + eval (cannot rely on dynamic import in non-module background scripts)
+            const code = await fetch(url).then(r => r.text());
+            (0, eval)(code); // explicit indirect eval
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)